src/main/java/no/sikt/nva/scrapers/DublinCoreOnlineValidator.java

```diff
@@ -0,0 +1,73 @@
+package no.sikt.nva.scrapers;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import no.sikt.nva.model.ErrorDetails;
+import no.sikt.nva.model.ErrorDetails.Error;
+import no.sikt.nva.model.dublincore.DcValue;
+import no.sikt.nva.model.dublincore.DublinCore;
+import nva.commons.doi.DoiValidator;
+import nva.commons.doi.UnitHttpClient;
+import org.jetbrains.annotations.NotNull;
+
```

Comment:
Kunne vi hatt DOI validering i en hel klasse for seg selv?
Altså dytte inn offline kallene også inn hit?

Reply:
implementert i egen klasse